docker/Dockerfile.contrib

```diff
@@ -26,23 +26,23 @@ RUN yarn install --network-timeout=${YARN_NETWORK_TIMEOUT}
 RUN yarn run build:full
 RUN yarn install --production --frozen-lockfile
 RUN for i in config core serial shared; do \
-    cd packages/$i && \
```

Comment:
I would prefer not reducing indentation spaces from 4 to 2.  It is also consistent with the other Dockerfile in this directory.

Reply:
> I would prefer not reducing indentation spaces from 4 to 2. It is also consistent with the other Dockerfile in this directory.

You're right - this was slipped through as well due to auto-format on save.